2021-06-04 11:53:48,204 INFO    MainThread:1280 [wandb_setup.py:_flush():69] setting env: {}
2021-06-04 11:53:48,204 INFO    MainThread:1280 [wandb_setup.py:_flush():69] setting login settings: {'anonymous': 'allow'}
2021-06-04 11:53:48,204 INFO    MainThread:1280 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_115348-312bxp10\logs\debug.log
2021-06-04 11:53:48,204 INFO    MainThread:1280 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\robertorg\Documentos\UPM\TFG\Aplicacion\wandb\run-20210604_115348-312bxp10\logs\debug-internal.log
2021-06-04 11:53:48,205 INFO    MainThread:1280 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001A1C49BDB08>
2021-06-04 11:53:48,206 INFO    MainThread:1280 [wandb_init.py:init():369] calling init triggers
2021-06-04 11:53:48,206 INFO    MainThread:1280 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-06-04 11:53:48,206 INFO    MainThread:1280 [wandb_init.py:init():418] starting backend
2021-06-04 11:53:48,207 INFO    MainThread:1280 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-06-04 11:53:48,221 INFO    MainThread:1280 [backend.py:ensure_launched():132] starting backend process...
2021-06-04 11:53:48,276 INFO    MainThread:1280 [backend.py:ensure_launched():137] started backend process with pid: 2744
2021-06-04 11:53:48,276 INFO    MainThread:1280 [wandb_init.py:init():423] backend started and connected
2021-06-04 11:53:48,277 INFO    MainThread:1280 [wandb_init.py:init():465] updated telemetry
2021-06-04 11:53:48,278 INFO    MainThread:1280 [wandb_init.py:init():484] communicating current version
2021-06-04 11:53:49,187 INFO    MainThread:1280 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-06-04 11:53:49,188 INFO    MainThread:1280 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-06-04 11:53:49,460 INFO    MainThread:1280 [wandb_init.py:init():522] starting run threads in backend
2021-06-04 11:53:53,608 INFO    MainThread:1280 [wandb_run.py:_console_start():1565] atexit reg
2021-06-04 11:53:53,609 INFO    MainThread:1280 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-06-04 11:53:53,609 INFO    MainThread:1280 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-06-04 11:53:53,609 INFO    MainThread:1280 [wandb_run.py:_redirect():1479] Redirects installed.
2021-06-04 11:53:53,610 INFO    MainThread:1280 [wandb_init.py:init():546] run started, returning control to user process
2021-06-04 11:54:05,740 INFO    MainThread:1280 [wandb_init.py:_pause_backend():249] pausing backend
2021-06-04 11:54:05,740 INFO    MainThread:1280 [jupyter.py:save_ipynb():265] not saving jupyter notebook
